src/webapp/components/template-selector/TemplateSelector.tsx

```diff
@@ -270,11 +273,12 @@ export const TemplateSelector = ({
         setState(state => ({ ...state, filterTEIEnrollmentDate }));
     };
 
-    const onFilterOrgUnitsChange = (_event: React.ChangeEvent, filterOrgUnits: boolean) => {
-        const isCustomProgram = state.templateType === "custom" && state.type !== "dataSets";
-        const populate = isCustomProgram && filterOrgUnits;
-        setState(state => ({ ...state, populate }));
-        clearPopulateDates();
+    const onFilterOrgUnitsChange = (filterOrgUnits: boolean) => {
+        //do not reset populate
+        // const isCustomProgram = state.templateType === "custom" && state.type !== "dataSets";
+        // const populate = isCustomProgram && filterOrgUnits;
+        // setState(state => ({ ...state, populate }));
+        // clearPopulateDates();
```

Comment:
Typically, commented code should be simply removed, we can use git to go back (We may do this for some temporal code, but not in general.)